<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGroup</class>
 <widget class="QWidget" name="CreateGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>7</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_bg" native="true">
     <widget class="QWidget" name="widget_h" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #09A3DC;</string>
      </property>
      <widget class="QPushButton" name="pushButton_close">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>0</y>
         <width>31</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
font: 87 9pt &quot;Segoe UI Black&quot;;</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>101</x>
         <y>4</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>创建群聊</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_mb" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>321</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QPushButton" name="pushButton_create">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>100</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);
font: 87 9pt &quot;Segoe UI Black&quot;;</string>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>231</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_gn">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>群名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_gn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>群聊名称</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>CreateGroup</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
